packages/twenty-front/vite.config.ts

```diff
@@ -166,6 +166,68 @@ export default defineConfig(({ command, mode }) => {
               return 'scalar';
             }
 
+            if (id.includes('react-dom')) {
+              return 'react-dom';
+            }
+
+            if (id.includes('react-router')) {
```

Comment:
I don't see a value to extract react-router, usually, we have a index.js containting the app skeleton (router, react, react-dom, apollo, ...) and chunk by page or by big component (like block editor or scalar)

Reply:
Not sure but it seems like it's faster with HTTP2 to parallelize

Reply:
I don't think either makes a big difference 🤔